app/src/main/kotlin/com/wire/android/ui/common/avatar/UserProfileAvatar.kt

```diff
@@ -201,6 +210,18 @@ fun UserProfileAvatar(
             val exactPointOnAvatarBorder = sqrt(2f) / 2f * ((size.value / 2f) + avatarBorderWidth.value)
             val maxOffset = (size.value / 2f) - (statusSize.value / 2f) - -statusBorderWidth.value
             val offsetToAlignWithAvatar = min(maxOffset, exactPointOnAvatarBorder)
+
+            if (shouldShowCreateTeamUnreadIndicator) {
+                UnreadInfoIndicator(
+                    modifier = Modifier
+                        .offset(x = offsetToAlignWithAvatar.dp, y = -offsetToAlignWithAvatar.dp)
+                        .onGloballyPositioned {
+                            userStatusIndicatorParams = it.size.toSize() to it.positionInParent()
+                        }
+                        .testTag(UNREAD_INFO_TEST_TAG)
```

Comment:
Question: Are we using this test tag at all 🤔 ? maybe we can remove it or better add a test for it 🙌

Reply:
it might be required later from QA